tamars_cookies.md

```diff
@@ -0,0 +1,42 @@
+# Ingredients
```

Comment:
If you look in the `challah.md` file, Ingredients should be lower case, use two `#`'s (meaning a 2nd tier header), and the name of the recipe should be at the top with a single `#`